stages/org.osbuild.lorax-script.meta.json

```diff
@@ -36,10 +40,20 @@
           "version": {
             "type": "string"
           }
-        },
-        "libdir": {
-          "type": "string",
-          "default": "lib64"
```

Comment:
This looks like a schema change.  `libdir` was under `product` but now it's at the top level.  Was this a mistake before?  From the code, it looks like it was.  Did we never use it?

It does look like it's a 4+ year old bug: 200c6c373c11f6837ac2564f7e1f7329c0232db4
Given the code reads libdir from the top-level `options` but the schema defined it under `product`, I suppose there was no way to specify `libdir` until now.

Anyway, could you please split this into its own commit so it's more visible and we have an atomic bugfix patch in the commit history?